front/app/modules/commercial/content_builder/admin/components/ContentBuilderToolbox/index.tsx

```diff
@@ -65,18 +65,103 @@ const ContentBuilderToolbox = ({
       borderRight={`1px solid ${colors.mediumGrey}`}
     >
       <Box w="100%" display="inline">
+        <Title mt="24px" ml="5px" variant="h6" as="h1" color="label">
+          <FormattedMessage {...messages.sections} />
+        </Title>
+        <DraggableElement
+          id="e2e-draggable-image-text-cards"
+          ref={(ref) =>
+            ref &&
+            connectors.create(
+              ref,
+              <Element id="image-text-cards" is={Box} canvas>
+                <TwoColumn columnLayout="1-2">
+                  <Element id="left" is={Container} canvas>
+                    <Image alt="" />
+                  </Element>
+                  <Element id="right" is={Container} canvas>
+                    <Text text={formatMessage(messages.textValue)} />
+                  </Element>
+                </TwoColumn>
+                <TwoColumn columnLayout="1-2">
+                  <Element id="left" is={Container} canvas>
+                    <Image alt="" />
+                  </Element>
+                  <Element id="right" is={Container} canvas>
+                    <Text text={formatMessage(messages.textValue)} />
+                  </Element>
+                </TwoColumn>
+                <TwoColumn columnLayout="1-2">
+                  <Element id="left" is={Container} canvas>
+                    <Image alt="" />
+                  </Element>
+                  <Element id="right" is={Container} canvas>
+                    <Text text={formatMessage(messages.textValue)} />
+                  </Element>
+                </TwoColumn>
+              </Element>
+            )
+          }
+        >
+          <ToolboxItem
+            icon="imageCardSection"
+            label={formatMessage(messages.imageTextCards)}
+          />
+        </DraggableElement>
+        <DraggableElement
+          id="e2e-draggable-info-accordions"
+          ref={(ref) =>
+            ref &&
```

Comment:
When I was asking Ben about the copy, he caught that the section was missing a whitespace, and that it should be separated into two TwoColumns (I think I looked only at the live version of the original design by accident) - so I've just updated that now.